website/docs/jellyfish/api/account.md

```diff
@@ -334,3 +336,55 @@ interface CommunityBalanceData {
   Unknown?: BigNumber
 }
 ```
+
+## listBurnHistory
+
+Returns information about burn history
+
+```ts title="client.account.listBurnHistory()"
+interface account {
+  listBurnHistory(
```

Comment:
If you look at other RPCs in the same file, there is always a space between` function name` and `(`
And, you wrote like this too in `account.ts`. See
<img width="692" alt="Screenshot 2021-07-19 at 10 22 22 PM" src="https://user-images.githubusercontent.com/7101806/126175633-e309623c-6207-4452-a866-17cdb1f09951.png">


Hence, change `listBurnHistory(` to `listBurnHistory (`